web/modules/custom/nfafmis/nfafmis.deploy.php

```diff
@@ -199,3 +200,78 @@ function nfafmis_deploy_004(&$sandbox = NULL) {
     }
   }
 }
+
+/**
+ * Initialise CFR summary data by adding area_summary entities.
+ */
+function nfafmis_deploy_005(&$sandbox = NULL) {
```

Comment:
how does this deploy.php file works, is this a drush deploy thing?

Reply:
Yes, it's for deploy hooks that are run after config has been imported. HOOK_post_update is called before drush cim and HOOK_deploy is called after drush cim.